sklearn/model_selection/tests/test_split.py

```diff
@@ -880,10 +882,20 @@ def test_cv_iterable_wrapper():
     # results
     kf_randomized_iter = KFold(n_splits=5, shuffle=True).split(X, y)
     kf_randomized_iter_wrapped = check_cv(kf_randomized_iter)
-    assert_array_equal(list(kf_randomized_iter_wrapped.split(X, y)),
-                       list(kf_randomized_iter_wrapped.split(X, y)))
-    assert_true(np.any(np.array(list(kf_iter_wrapped.split(X, y))) !=
-                       np.array(list(kf_randomized_iter_wrapped.split(X, y)))))
+    # numpy's assert_array_equal properly compares nested lists
```

Comment:
@jnothman would this suffice? or would you prefer having this imported into `sklearn.utils.testing` rather?